gtda/homology/simplicial.py

```diff
@@ -1809,7 +1809,7 @@ def _lower_star_extended_diagram(self, X):
         data_diag = np.zeros(n_points_cone, dtype=X.dtype)
         data_diag[:n_diag] = X.diagonal()
         max_value = data_diag[:n_diag].max()
-        min_value = data_diag[:n_diag].min()
+        min_value = data_diag[:n_points].min()
```

Comment:
Why this?

Reply:
I mention it briefly [here](https://github.com/giotto-ai/giotto-tda/pull/561#discussion_r575831375).
Take a function on a simplex with 3 vertices: [4, 3, (0) ], the last being an implicit 0. Then, `n_diag=2` and `min_value=3`. However, there is one vertex appearing at 0, while `data_diag[-1] = 2`.